File: CODE FILES/Backend/MyApiary/Resource.Api/Models/CreateSession.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Resource.Api.Models
{
    public class CreateSession
    {
        public int Device_id { get; set; }

        [Required]
        public string Device_Name { get; set; }

        public string Device_Mac_address { get; set; }

        [Required]
        public string Device_Fingerprint { get; set; }

        [Required]
        public string Device_Refresh_token { get; set; }

        public int Session_Id { get; set; }

        public string Session_Ip_address { get; set; }

        public string Session_Geolocation { get; set; }
    }
}

File: CODE FILES/Mobile/MyApiary/app/src/main/java/projects/myapiary/models/CreateSession.kt
package projects.myapiary.models

data class CreateSession(
    val Device_id: Int,
    val Device_Name: String,
    val Device_Mac_address: String,
    val Device_Fingerprint: String,
    val Device_Refresh_token: String,
    val Session_Id: Int,
    val Session_Ip_address: String,
    val Session_Geolocation: String
)
